sdks/java/io/amazon-web-services2/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
   compile library.java.aws_java_sdk2_sdk_core
   compile library.java.aws_java_sdk2_sns
   compile library.java.aws_java_sdk2_sqs
+  runtime library.java.aws_java_sdk2_sts
```

Comment:
Nit: let's list different types of dependencies (`compile`, `testCompile`, etc.) contiguously so it's easier to read.